src/main/java/Main.java

```diff
@@ -0,0 +1,172 @@
+import spark.Spark;
+import spark.ModelAndView;
+import spark.Session;
+import spark.template.mustache.MustacheTemplateEngine;
+
+import java.util.*;
+
+/**
+ * Created by vasantia on 7/13/16.
+ */
+public class Main {
+
+    public static final String SESSION_USER = "userName";
+    static Map<String, User> users = new HashMap<>();
+    static List<Bottle> allBottles = new ArrayList<>();
+
+    public static void main(String[] args) {
+
+        Spark.staticFileLocation("public");
+        Spark.init();
+
+        Spark.get("/", (request, response) -> {
+            Map m = new HashMap();
+            Session session = request.session();
+            String name = session.attribute(SESSION_USER);
+            User user = users.get(name);
+
+//            List<Bottle> bottlePages = new ArrayList<>();
```

Comment:
Please remember to remove commented out code before final submission.  It makes the code look sloppier than it is in reality.